rust/README.md

```diff
@@ -8,8 +8,9 @@ This UDC runs the cargo command `cargo $args` caching the contents of `$CARGO_HO
 
 ### Usage
 
-First, import the UDC up in your Earthfile:
+First, import the UDC up in your Earthfile. Also, the UDC makes use of global caches, so make sure to enable the feature flag as follows:
```

Comment:
Hmm.. is this strictly necessary on the user's side? It might be a bug in the way feature flags are interpreted. It should only be necessary in the file where the feature is actually used.

Reply:
Actually it is not, thanks!